lib/memjs/memjs.js

```diff
@@ -355,7 +356,7 @@ Client.prototype.flush = function(callback) {
     serv.write(seq, request);
   };
 
-  for (var i in this.servers) {
+  for (i = 0; i < this.servers.length; i++) {
```

Comment:
So JS linters and style guides recommend you declare variables at the top of functions since JS only has function scope, not block scope. And so declaring like old school C makes this obvious.

Thoughts?